parquet-tools/src/main/java/org/apache/parquet/tools/Main.java

```diff
@@ -162,7 +162,8 @@ public static void die(String message, boolean usage, String name, Command comma
   }
 
   public static void die(Throwable th, boolean usage, String name, Command command) {
-    die(th.getMessage(), usage, name, command);
+    String message = th.getClass().getSimpleName() + ": " + th.getMessage();
```

Comment:
I added name + message to make clear it is a failure in cases when exception's `toString()` is overwritten to return just message.